ui/core/src/actions/clp/index.ts

```diff
@@ -139,6 +139,11 @@ export default ({
       wBasisPoints: string,
       asymmetry: string
     ) {
+
+      // Note: Chain needs -1 - 1, 
+      // UI needs to do -10000 - 10000 bc current Fraction floating point limitation
+      asymmetry =  (+asymmetry/10000).toString()
```

Comment:
Wow I didn't know you could do that! Is that JS type coercion?

Reply:
haha yea